suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -362,7 +362,7 @@ export const formatAmount = (amount: BigNumberValue, decimals: number) => {
     }
 };
 
-export const amountToSatoshi = (amount: BigNumberValue, decimals: number) => {
+export const formatAmountWithDecimals = (amount: BigNumberValue, decimals: number) => {
```

Comment:
Just a nitpick, but I think the new name makes it unclear what this function does. 
It's named confusingly similarly to `formatAmount` function, but it does the opposite thing.
So maybe `amountToSmallestUnit` or `normalizeAmount`?

Reply:
It is clear for me, but I will rename it. np